src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private ColorSupplier loteria = new ColorSupplier();
+    private Random randomValueSupplier = new Random();
+
+    public Ball getRandomBall() {
+        int number = randomValueSupplier.nextInt(101);
```

Comment:
The range for 'nextInt' should be extracted to a constant field according to the checklist. This will improve code readability and make it easier to manage the range value.